src/components/pick-list-item/pick-list-item.e2e.ts

```diff
@@ -127,4 +127,30 @@ describe("calcite-pick-list-item", () => {
 
     expect(removeEventSpy).toHaveReceivedEventTimes(1);
   });
+
+  it("slot keyboard navigation", async () => {
+    const page = await newE2EPage({
+      html: `<calcite-pick-list style="width: 400px">
```

Comment:
Can you use the html test util so that embedded code is automatically formatted by Prettier?